### ISSUE TITLE
Divide the study more into components

### ISSUE THREAD

**Issue by EllaSoderberg** [EllaSoderberg/EmotionandLiking#9]
Now there is a lot of copy-paste required which is very time-consuming. I couldn't figure out how to loop the blocks on the same components so now it kind of sucks to edit this whole thing :( So if possible it'd be awesome to divide it into components where edits would affect all blocks at once instead of just one at a time.

**Comment by Geoodees** [comment]
I would say this is just an issue with LabJS... Make sure everything is done before creating all components.

**Comment by EllaSoderberg** [comment]
Yeah.. But also, if we had the file names be the same apart from the initial letter, we'd be able to create one loop per part instead of three and four! But feels like too much of a hassle to figure out at this point...
